src/terminal/pty/UnixPty.h

```diff
@@ -21,6 +21,7 @@
 
 #if defined(__APPLE__)
 #include <util.h>
+#elif defined(__FreeBSD__)
```

Comment:
Empty elif-branch can be eliminated.

Reply:
Issue is that pty.h does not exist on FreeBSD. If it is only a thing on Linux, then we may wanna change this into a ifdef for Linux.